packages/mdc-snackbar/index.js

```diff
@@ -47,6 +47,16 @@ export class MDCSnackbar extends MDCComponent {
       unsetActionAriaHidden: () => getActionButton().removeAttribute('aria-hidden'),
       setActionText: (text) => { getActionButton().textContent = text; },
       setMessageText: (text) => { getText().textContent = text; },
+      setFocus: () => getActionButton().focus(),
+      visibilityIsHidden: () => document.hidden,
+      registerBlurHandler: (handler) => getActionButton().addEventListener('blur', handler, true),
```

Comment:
Okay. That makes sense. I'd rather have it be explicitly clear, rather than rely on the signature for `addEventListener`